src/Razor/test/VSCode.FunctionalTest/tests/index.ts

```diff
@@ -0,0 +1,103 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as fs from 'fs';
+import * as Mocha from 'mocha';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+// This file controls which tests are run during the functional test process.
+export async function run(): Promise<void> {
+    const mocha = new Mocha({
+        ui: 'tdd',
+        timeout: 60000,
```

Comment:
Curious: This is during each test and non-inclusive of the `before` actions we have where we restore right?

Reply:
Yeah this timeout is for a single test. I have explicitly overridden this to 5 minutes in the `before()` of `Main.test.ts`.